.github/workflows/deployTest.yml

```diff
@@ -8,28 +8,33 @@ env:
   ARM_CLIENT_SECRET: ${{ secrets.TERRAFORM_ARM_CLIENT_SECRET }}
   ARM_SUBSCRIPTION_ID: ${{ secrets.TERRAFORM_ARM_SUBSCRIPTION_ID }}
   ARM_TENANT_ID: ${{ secrets.TERRAFORM_ARM_TENANT_ID }}
-  ACR_ADMIN_USERNAME: ${{ secrets.ACR_ADMIN_USERNAME }}
-  ACR_ADMIN_PASWORD: ${{ secrets.ACR_ADMIN_PASWORD }}
-  ACR_REPO_URL: ${{ secrets.ACR_REPO_URL }}
```

Comment:
This is a slightly unrelated "we don't need those to be in the environment, do we?" tweak.